dbsc/workflow/rules/mrtpipelines.smk

```diff
@@ -364,6 +371,18 @@ rule tckgen:
     params:
         step=config["step"],
         sl=config["sl_count"],
+        tmp_dir=bids_tractography_out(
+            root=os.environ.get("SLURM_TMPDIR")
+            if config.get("slurm_tmpdir")
+            else "/tmp",
+        ),
+        tmp_tck=bids_tractography_out(
+            root=os.environ.get("SLURM_TMPDIR")
```

Comment:
The `os.environ.get("SLURM_TMPDIR") if config.get("slurm_tmpdir") else "/tmp"` expression comes up often enough that it probably makes sense to assign it to a variable or add it to the config dict (i.e. something like `tmpdir` or `config["tmpdir"]` at the top of this file.

Reply:
From testing, this doesn't seem to work when assigned to a variable. Left this part of the code as is, but moved from resources to params.

Reply:
Optional: Could try making it a function instead of assigning it to a variable i.e.

```
def get_tmpdir():
    return os.environ.get("SLURM_TMPDIR") if config.get("slurm_tmpdir") else "/tmp"
```